File: reverseNum.java
public class reverseNum{
    public static void main(String args[]){
        int num = 121;
        int orginalNum=num;
        int rev = 0;
        for(int i =0; i<3; i++){
            int temp = num%10;
            rev=rev*10+temp;
            num = num/10;
        }
        System.out.println(rev==orginalNum);
    }
}

File: reverseNum.go
package main

import "fmt"

func main() {
	var num int = 121
	var orginalNum int = num
	var reverse int = 0

	for i := 0; i < 3; i++ {
		var temp int = num % 10
		reverse = reverse*10 + temp
		num = num / 10
	}
	fmt.Println(reverse == orginalNum)

}
